cli/src/test/kotlin/de/deutschepost/sdm/cdlib/archive/ArchiveCommandIntegrationTest.kt

```
@@ -171,9 +172,12 @@ class ArchiveCommandIntegrationTest(@Value("\${artifactory-azure-identity-token}
     override suspend fun beforeTest(testCase: TestCase) {
         super.beforeTest(testCase)
         val repository = artifactory.repository(repoName)
+        val logger = mu.KotlinLogging.logger {}
+        
         try {
             repository.delete(releaseName_build)
-        } catch (_: Exception) {
+        } catch (exception: Exception) {
+            logger.error(exception) { "Failed to delete repository folder: $releaseName_build" }
```

To address the issue of an empty catch block and improve error tracking, I have added a logging statement using KotlinLogging, a library available in the existing dependencies (io.github.microutils:kotlin-logging). This logger will log the exception details at the ERROR level, which is appropriate for exceptions that occur within this context, ensuring that any runtime issues are not silently ignored. The logger is initialized at the beginning of the code section within the allowed range. This change also adheres to the guidelines of using an appropriate logging framework without adding any new import statements.<br><br>[Related Sonar Issue](http://localhost:9000/issues?open=15156638-4782-4aec-a774-3f852ff7c2ca)<br>[Related Sonar Rule](http://localhost:9000/coding_rules?open=kotlin:S108&rule_key=kotlin:S108)